src/app/dataset-block/metadata-block/components/event-details/supported.events.ts

```diff
@@ -10,3 +10,8 @@ export enum SupportedEvents {
     SetVocab = "SetVocab",
     SetWatermark = "SetWatermark",
 }
+
+export interface LogoInfo {
```

Comment:
It's either a wrong file to define `LogoInfo`, or the name of the file is wrong. Currently it's suggesting we focus on supported events. Logo is not about events.